implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -221,6 +208,23 @@ static double minFromHistogramSnapshot(HistogramSnapshot histogramSnapshot, Time
         return streamOf(serializedLine);
     }
 
+    private Stream<String> toSummaryLine(Meter meter, HistogramSnapshot histogramSnapshot, double total, double max, boolean isTimer) {
```

Comment:
If the user registered their own histogram snapshots in addition to our 0.0 one, will this be called multiple times?
Please add a test verifying that this works in these cases and only exports once with the correct min value.